doc/CONFIGURE.md

```diff
@@ -200,12 +201,12 @@ and the specified hostname and port, so where you have created the appropriate c
 will be restricted to those outside of any credential domain and those from matching credential domains.
 This can help differentiate between multiple keys/password associated with the same username.
 
-## Using Cygwin
-See [SSH slaves and Cygwin](https://wiki.jenkins.io/display/JENKINS/SSH+slaves+and+Cygwin) for the discussion of how to use this plugin to talk to Cygwin SSHD server.
+### Using Cygwin
```

Comment:
@kuisathaverat I am thinking about moving this Wiki to plugin docs . Do we really want to do it? Or would you rther prefer to deprecate it in favor of native SSH support by modern Windows versions?

Reply:
I  guess that old windows versions still do not have native SSH support, so this documentation is still valid, thus we should move it to plugin documentation.

Reply:
OK, will do it in a follow-up pull request